game/src/contrib/components/CollideComponent.java

```diff
@@ -14,26 +14,54 @@
 
 import java.util.logging.Logger;
 
+/**
+ * Allows an entity to collide with other entities that have a {@link CollideComponent}.
+ *
+ * <p>The component creates a hitbox (an invisible square) around the associated entity.
```

Comment:
Wie groß ist das Square bzw. der Radius? Oder ist das einstellbar?

Reply:
square ist vielleicht falsch da es mehr ein rechteck ist. und kann über size bestimmt werden.

Reply:
dazu sollte ein kurzer nachsatz in die javadoc :)